torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -11,6 +11,12 @@
 from torch.ao.quantization.qconfig import QConfig
 from torch.ao.quantization.quantize import is_activation_post_process
 
+# Names for observer insert keys
+DETECTOR_TARGET_NODE_KEY = "target_node"
+DETECTOR_OBSERVER_INSERT_KEY = "observer_to_insert"
+DETECTOR_POST_KEY = "is_post_observer"
```

Comment:
nit: maybe just spell these out to make it clearer, e.g.
'DETECTOR_OBS_TO_INSERT_KEY' and 'DETECTOR_IS_POST_OBS_KEY'